news/2021-01-14-performance-points-updates.md

```diff
@@ -0,0 +1,127 @@
+
+## Spun Out Changes
+
+Similarly to how NoFail used to function as described above, Spun Out also applied a flat 5% reduction in the maximum possible pp (again, not score) you could receive while making a play with it enabled.
+
+As suggested by [**StanR**](https://osu.ppy.sh/users/7217455) in [this pull request](https://github.com/ppy/osu-performance/pull/110), the following change has been applied:
+
+- Spun Out's pp multiplier now scales based on the amount of spinners in the map.
+    - New value: ```1 - (Number of Spinners / Total Hitobjects)^0.85```
+    - Old value: 0.95x (always)
+
+Spun Out should now punish as much as it has an actual impact on the general difficulty of a map, which is to say *not very much*. This is mostly a high-end change, as lower difficulty maps with more meaningful spinners will still require newer players to learn to spin properly.
+
+This also fixes an issue where Spun Out reduced pp on maps with no spinners present if enabled, even though it technically affected nothing.
+
+## Miss Penalty Curve Adjustments
+
+To understand these changes, it is important to know that the performance points algorithm previously penalized misses *exponentially*, reducing the aim and speed pp components awarded for a score based on a simple function that factored in only the number of misses made during a play and absolutely nothing else.
+
+This meant that 3 misses on a 100 note map was functionally equivalent to 3 misses on a 1000 note map.
+
+As suggested by [**Xexxar**](https://osu.ppy.sh/users/2773526), the following change has been applied:
+
+- The miss penalty curve has been adjusted.
+    - Any number of misses on a map instantly incurs a once-off 3% penalty to total pp awarded for both aim and speed.
+    - The first few misses are penalised less harshly than the ones after. The largest relative punishment occurs somewhere around the 15% miss mark (relative to the number of objects in the map), after which point the reduction for each subsequent miss tapers off again.
+    - Old value: Aim and speed values reduced by ```0.97^x``` where x equals the number of misses.
+
+To help illustrate these changes better, consult the graph image below. The green line refers to the old formula, and the purple line refers to the new one. You can also view this graph live and play around with the settings following [this link](https://www.desmos.com/calculator/rshij9757a).
+
+![](/wiki/shared/news/2021-01-14-performance-points-updates/miss-penalty-curve.png)
+
+This change seeks to penalize misses on short maps more, while significantly reducing the impact of a small number of misses on longer maps. The one-off 3% penalty for a miss provides incentive for players to hone any 1-miss plays they might still make, but otherwise makes them considerably less punishing on longer plays.
```

Comment:
```suggestion
This change seeks to penalize misses on short maps more, while significantly reducing the impact of a small number of misses on longer maps. The one-off 3% penalty for a miss provides an incentive for players to hone any 1-miss plays they might still make but otherwise makes them considerably less punishing on longer plays.
```
missing determiner and unnecessary comma usage

Reply:
`an` isn't required here. comma is there for pacing. i think it's fine as-is.